File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {

        int[] arr = new int[26];

        for(char i : s.toCharArray()){
            arr[i- 'a']++;
        }
        
        for(char i : t.toCharArray()){
            arr[i- 'a']--;
        }

        for(int i=0;i<arr.length;i++){
            if(arr[i] !=0) return false;
        }

        return true;
    }
}

File: 0242-valid-anagram/0242-valid-anagram.js
    var isAnagram = function(s, t) {



    let arr = new Array(26).fill(0);

    for (let char of s) {
        arr[char.charCodeAt(0) - "a".charCodeAt(0)]++;
    }

    for (let char of t) {
        arr[char.charCodeAt(0) - "a".charCodeAt(0)]--;
    }

    for (let num of arr) {
        if (num !== 0) return false;
    }

    return true;

    };
